### ISSUE TITLE
MP4 support instead of .roq

### ISSUE THREAD

**Issue by dinurdo** [eternalcodes/EternalJK#48]
Title

**Comment by videoP** [comment]
good idea, .roq is ancient and takes up way too much space 

mp4 is modern and already used in modern engines
https://docs.unrealengine.com/latest/INT/Engine/MediaFramework/HowTo/FileMediaSource/

**Comment by srcanseco** [comment]
Yes, they have a place in multiplayer map making where they can be used into shaders where needed. Maybe not the most common thing, and maybe bad practice fps-wise, but it can be used.
